webcompat/templates/issue/issue-header.html

```diff
@@ -0,0 +1,20 @@
+{% from "web_modules/tag.html" import tag with context %}
+
+<header class="wc-IssueDetail">
+  <h1 class="wc-IssueDetail-title">
+    <script type="text/template" id="title-tmpl">
+    Issue <%= number %>: <%- title %>
+    </script>
+  </h1>
+  <!-- Issue Date -->
+  <div class="wc-IssueDetail-create">
+    <script type="text/template" id="metadata-tmpl">
+    <div class="wc-Tag wc-Tag--label wc-Tag--<%= stateClass %>"><%= issueState %></div>
```

Comment:
Hmm, I think we might be able to get around the templating error with the `{%raw%} blah {%endraw%}` in the macro definition: http://jinja.pocoo.org/docs/dev/templates/#escaping

But maybe it won't work because stateClass is defined in JS and you're hitting a reference error before the macro is even run.

`{{ tag('filter', <%= stateClass %>) }}` maybe possible to define in such a way to do `{{ tag('filter', 'stateClass') }}` (pass in stateClass as a string)?